docsite/rst/playbooks_filters.rst

```diff
@@ -316,28 +338,32 @@ To get the directory from a path::
 
     {{ path | dirname }}
 
-To expand a path containing a tilde (`~`) character (new in version 1.5)::
+To expand a path containing a tilde (`~`) character (since 1.5)::
```

Comment:
ack'd, I wasn't sure, as there were some with 'since' and some with 'new in', so i went with the shorter option. will update.